batch-service/src/main/java/com/revature/batch/controller/BatchController.java

```diff
@@ -1,214 +1,54 @@
 package com.revature.batch.controller;
 
-import java.sql.Timestamp;
-import java.util.List;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.revature.batch.bean.Batch;
-import com.revature.batch.bean.BatchType;
-import com.revature.batch.exception.BatchUpdateException;
 import com.revature.batch.exception.NoBatchException;
 import com.revature.batch.service.BatchService;
-import com.revature.batch.service.TrainerService;
 
+/**
+ * 
+ * @author Josh Boudreau, Sonam Sherpa, Marko Miocic
+ * Batch: 1802-Feb12-java-matt
+ * 
+ * handles ZUUL endpoint /batches
+ *
+ */
 @RestController
+@RequestMapping(value = "/batch")
```

Comment:
so full url for your controller from gateway to this controller is gonna be "localhost:9001/api/v2/batches/batch"? Is this what you want it to be? - OctoDad

Reply:
Yes, because all appending values will relate to a single batch

Reply:
sweet, thanks for clarifying. 🍬